### ISSUE TITLE
Host a live demo

### ISSUE THREAD

**Issue by ebugden** [theia-ide/theia-trace-extension#173]
Hosting a live demo of the trace extension using a Theia-based IDE completely removes the setup time so it greatly encourages people to try out the tool.

* Removing the setup makes collecting feedback from busy analysis experts much more realistic
* Example: [Grafana live demo](https://play.grafana.org/) 

Prerequisite: Allow connecting to a remote trace server (by specifying the URL #79)

**Comment by tahini** [comment]
A related issue that may give a nice first live demo on gitpod: https://github.com/theia-ide/theia-trace-extension/issues/79

**Comment by ebugden** [comment]
An additional way of providing a demo would be to create a minimal single-purpose Theia application only for trace viewing: 
* Issue #84: Add minimal trace-viewer example app

**Comment by ebugden** [comment]
The README will soon explain how to run a demo via Gitpod so this issue can be "put to sleep" for the next couple months. Having a Gitpod live demo meets the project's current needs since people who are trying the tool at this stage are potentially also interested in seeing the source code. 

Further in the future, when people trying out the tool are less involved in development, it could become valuable to provide a live demo that goes straight to the tool similar to Grafana.

**Comment by ebugden** [comment]
Right now there's a [live demo](https://gitpod.io/#https://github.com/theia-ide/theia-trace-extension) via Gitpod and if we want something directed more towards users than devs we can create a separate issue for that.
